include/gatekeeper_gt.h

```diff
@@ -202,6 +203,8 @@ struct gt_cmd_entry {
 
 	union {
 		lua_State *lua_state; /* GT_UPDATE_POLICY */
+		/* GT_UPDATE_POLICY_INCREMENTALLY */
+		char lua_bytecode_filename[128];
```

Comment:
Instead of passing a filename for `GT_UPDATE_POLICY_INCREMENTALLY`, we should pass a memory block (i.e. `rte_malloc()`) with the byte code. Once the byte code is executed, the GT block releases the memory. Therefore, each GT block will receive a copy of the byte code to execute.

Reply:
Indeed, the first version that I came up was to transfer the memory block directly, however, there is an error message: ``malformed bytecode`` that I didn't get rid of it.